crates/uv-fs/src/which.rs

```diff
@@ -1,5 +1,26 @@
 use std::path::Path;
 
+#[cfg(target_os = "windows")]
```

Comment:
```suggestion
#[cfg(windows)]
```

(Either is fine, we use this one more)